cluster/gce/gci/configure-helper.sh

```diff
@@ -48,16 +48,32 @@ function create-dirs {
   fi
 }
 
+# Formats the given device ($1) if needed and mounts it at given mount point
+# ($2).
+function safe-format-and-mount() {
+  device=$1
+  mountpoint=$2
+
+  # Format only if the disk is not already formatted.
+  if ! tune2fs -l "${device}" ; then
```

Comment:
The 'blkid' test is just to ensure that the device is a block device. The main command used here to check filesystem format is the `file` command which is not present on GCI. So that code isn't working as expected today anyways.
'tune2fs' takes care of FS format check for us. Since we only support EXT4 filesystem on GCI, this is sufficient. Other FS like XFS are not supported. So tune2fs is sufficient.